backend/src/main/java/com/praksa/auction/controller/ProductController.java

```diff
@@ -0,0 +1,41 @@
+package com.praksa.auction.controller;
+
+import com.praksa.auction.model.Product;
+import com.praksa.auction.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@CrossOrigin
+@RestController
+@RequestMapping("/api")
+public class ProductController {
+
+        private final ProductService productService;
+
+        @Autowired
+        public ProductController(ProductService productService) {
+            this.productService = productService;
+        }
+
+    @GetMapping("/productRandom")
+    public ResponseEntity<Product> getRandomProduct() {
+            var value = productService.getOneRandom();
+            return ResponseEntity.ok(value);
+    }
+
+    @GetMapping("/lastChance")
+    public ResponseEntity<List<Product>> getLast(@RequestParam int start, @RequestParam int count) {
+        var value = productService.getLastChance(start,count);
+        return ResponseEntity.ok(value);
+    }
+
+    @GetMapping("/newest")
+    public ResponseEntity<List<Product>> getNewest(@RequestParam int start, @RequestParam int count) {
+        var value = productService.getNewest(start,count);
+        return ResponseEntity.ok(value);
+    }
```

Comment:
Please use auto formatting. that Intelij offers on each file.
<img width="348" alt="Screenshot 2022-07-08 at 11 44 04" src="https://user-images.githubusercontent.com/48674642/177965192-3efd94f7-4752-488b-982a-587861e5d094.png">